website/_config.yml

```diff
@@ -36,6 +42,7 @@ social:
     - https://github.com/PicnicSupermarket
     - https://twitter.com/picnic
     - https://www.linkedin.com/company/picnictechnologies
+
```

Comment:
I'd omit this empty line, because the `twitter` section below is also part of the SEO config mentioned above.